src/main/java/org/jitsi/jigasi/JigasiBundleActivator.java

```diff
@@ -54,6 +54,30 @@
 
     public static BundleContext osgiContext;
 
+    /**
+     * The property name for the boolean value whether transcription should be
+     * enabled.
+     */
+    public final static String P_NAME_ENABLE_TRANSCRIPTION
+        = "org.jitsi.jigasi.enableTranscription";
```

Comment:
Can we rename to ENABLE_TRANSCRIPTION (and ENABLE_SIP) for consistency with the other properties that we have?